src/Program.cs

```diff
@@ -48,27 +48,14 @@ static async Task<int> Main(string[] args)
 
             var credential = await authServiceFactory.GetTokenCredentialAsync(authStrategy, authSettings?.TenantId, authSettings?.ClientId);
             var authProvider = new AzureIdentityAuthenticationProvider(credential, new string[] {"graph.microsoft.com"});
-            var defaultHandlers = KiotaClientFactory.CreateDefaultHandlers();
 
             var assemblyVersion = Assembly.GetExecutingAssembly().GetName().Version;
-            var sdkVersionHeaderValue = string.Format(
-                SdkVersionHeaderValueFormatString,
-                "graph",
-                assemblyVersion.Major,
-                assemblyVersion.Minor,
-                assemblyVersion.Build);
-
-            var telemetryHandlerOption = new TelemetryHandlerOption {
-                TelemetryConfigurator = (request) => {
-                    request.Headers.Add("SdkVersion", sdkVersionHeaderValue);
-                    return request;
-                }
+            var options = new GraphClientOptions {
+                GraphProductPrefix = "graph-cli",
+                GraphServiceLibraryClientVersion = $"{assemblyVersion.Major}.{assemblyVersion.Minor}.{assemblyVersion.Build}"
```

Comment:
```suggestion
                GraphServiceLibraryClientVersion = $"{assemblyVersion.Major}.{assemblyVersion.Minor}.{assemblyVersion.Build}",
```

Reply:
What's the point of this? 😅

Reply:
Code doesn't compile. Missing comma.

Reply:
Oh, right...I thought it was a trailing comma